src/store/getters.ts

```diff
@@ -1,71 +1,82 @@
 import { CorpTypeCd, FilingStatus, FilingTypes, EntityStatus } from '@/enums'
+import { StateIF } from '@/interfaces'
 
 export default {
   /** The current year. */
-  currentYear (state): number {
+  currentYear (state: StateIF): number {
     return state.currentDate ? +state.currentDate.substring(0, 4) : 0
   },
 
-  /** Is True if there is any pending tasks. */
-  hasBlockerTask (state): boolean {
-    return state.hasBlockerTask
+  /** Is True if there are any pending tasks or filings. */
+  hasBlocker (state: StateIF): boolean {
+    return (state.hasBlockerTask || state.hasBlockerFiling || state.isCoaPending)
```

Comment:
Technically, if `isCoaPending` is True then `hasBlockerFiling` must also be True, but I added it here anyway due to some unit tests that only set "isCoaPending=true" and expect `hasBlocker` to be True.